lib/oli_web/live/delivery/instructor_dashboard/instructor_dashboard_live.ex

```diff
@@ -189,6 +189,40 @@ defmodule OliWeb.Delivery.InstructorDashboard.InstructorDashboardLive do
     {:noreply, assign(socket, params: params, view: :overview, active_tab: :recommended_actions)}
   end
 
+  def handle_params(
+        %{"view" => "overview", "active_tab" => "practice_activities"} = params,
+        _,
+        socket
+      ) do
+    socket =
+      socket
+      |> assign(
+        params: params,
+        view: :overview,
+        active_tab: :practice_activities
+      )
+      |> assign_new(:students, fn ->
+        Sections.enrolled_students(socket.assigns.section.slug)
+        |> Enum.reject(fn s -> s.user_role_id != 4 end)
+      end)
+      |> assign_new(:practice_activities, fn %{students: students} ->
+        Helpers.get_assessments(socket.assigns.section, students)
```

Comment:
The `:practice_activities` assign is being populated with a call to `Helpers.get_assessments` which returns the **graded** pages.  Practice activities should be populated with the opposite set, those pages that are **not** graded.